docs/getting-started.md

```diff
@@ -16,15 +16,15 @@ The API will reject you if you don't provide a `User-Agent` header (more details
 
 If you want to access private repositories or perform actions on behalf of a user, you need to pass credentials to the client.
 
-There are two options supported by the API - basic and OAuth authentication.
+There are two Authentication options supported by the API. 
 
-```csharp
-var basicAuth = new Credentials("username", "password"); // NOTE: not real credentials
-client.Credentials = basicAuth;
-```
+- Personal Access Tokens (PAT) and 
+- OAuth authentication.
+
+If you wish to use a PAT follow this [link](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) to generate one, then replace "token" in the snippet below with the text generated by GitHub.
```

Comment:
Please could you remove the /en from the url? That way the GitHub website can work out what language it should display for the user 😄

Reply:
Found two or three instances and removed them.